Formula/zpython.rb

```diff
@@ -18,10 +18,14 @@ def message
 class Zpython < Formula
   desc "Embeds a Python interpreter into zsh"
   homepage "https://bitbucket.org/ZyX_I/zsh"
+  version "00-5.0.5"
```

Comment:
This should live in the `stable do` block, underneath the `url` or `mirror` where one exists.